### PR TITLE
Add Services API

### PR DESCRIPTION
**Pull request by nicl**
Just a 'Hello, world' really atm - adds a healthcheck.

Keen to add Google Auth but want to do this by extending the CDK pattern rather than bespoke here (which is verbose).

### CODE REVIEW

File: packages/cdk/lib/services-api.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+systemctl start ${app}
+`;
+
+		const ec2 = new GuEc2App(this, {
+			app,
+			access: {
+				scope: AccessScope.INTERNAL,
+				cidrRanges: [Peer.ipv4('10.0.0.0/8')], // TODO replace with Google auth once integrated to this pattern.
+			},
+			instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.NANO),
```
Comment (kenoir) [inline packages/cdk/lib/services-api.ts:61]: 🐜 💪

File: packages/services-api/src/app.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+			credentials: true,
+		}),
+	);
+
+	router.get('/healthcheck', (req: express.Request, res: express.Response) => {
+		res.status(200).json({ status: 'OK', stage: 'INFRA' });
+	});
+
+	//handle all invalid routes by showing all available routes
+	router.get('*', getDescribeRouterHandler(router));
```
Comment (kenoir) [inline packages/services-api/src/app.ts:25]: Not for here, but I reckon the `getDescribeRouterHandler` should `404` while still dispensing the routes in the body.

**Review by kenoir — APPROVED** [review]

File: packages/services-api/src/app.ts (hunk 2 above)
Comment (kenoir) [reply packages/services-api/src/app.ts:25]: https://github.com/guardian/github-lens/pull/42

File: packages/services-api/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "description": "API for Guardian services",
+  "scripts": {
+    "test": "jest --detectOpenHandles --config ../../jest.config.js --selectProjects services-api",
+    "prebuild": "rm -rf dist",
+    "build": "esbuild src/handler.ts --bundle --outdir=dist --platform=node --target=node16 --external:aws-sdk",
+    "dev": "LOCAL=true ts-node ./src/handler",
+    "start": "npm run dev",
+    "typecheck": "tsc -noEmit"
+  },
+  "devDependencies": {
```
Comment (akash1810) [inline packages/services-api/package.json:13]: Some/most of these devDeps are common across the other workspaces, and shouldn't need repeating. Not a blocker, but we should revise how things are setup, pushing as much to the root package.json, if only to have a simpler version bumping story.

**Pull request merged**